material.h

```diff
@@ -0,0 +1,56 @@
+/* 
+ @file      material.h
+ @brief     contains the material class
+ @author    Luke Eure
+ @date      January 6 2016
+*/
+
+#ifndef MATERIAL_H
+#define MATERIAL_H
+
+#include <iostream>
+#include <vector>
+
+class Material {
```

Comment:
Looking over this class, we seem to be using it simply as a struct and the data isn't really opaque. A better solution if we still want to use a class is to get cross sections given a group ie:
``getSigmaA(int group)``